### ISSUE TITLE
Ctrl-P not go line up problem in ACE key binding

### ISSUE THREAD

**Issue by reorx** [HappenApps/Quiver#272]
This is a known issue in ace editor: https://github.com/ajaxorg/ace/issues/2286 ,
which could be fixed by adding `editor.commands.bindKey("Ctrl-P", "golineup")` to manually bind it,
it would be very appreciated if you can also fix this problem in Quiver, since Ctrl-x (x to be B, F, P, N, etc)  is the de facto standard for moving cursor in a text area.

**Comment by ghost** [comment]
I also had met the problem like yours.

**Comment by burke** [comment]
You can patch it like this:

```
sed -i '' 's/n;e.bindKeys/n;e.bindKey("Ctrl-P","golineup");e.bindKeys/' /Applications/Quiver.app/Contents/Resources/html-build/js/editor.js
```

**Comment by ghost** [comment]
@burke thank u, it works for me!

**Comment by ylian** [comment]
Fixed in 3.0.1.

**Comment by reorx** [comment]
@ylian Thank you!
